nexus/src/authz/omicron.polar

```diff
@@ -423,14 +444,9 @@ has_permission(actor: AuthenticatedActor, "read", device_token: DeviceAccessToke
 
 has_permission(actor: AuthenticatedActor, "read", identity_provider: IdentityProvider)
 	if has_role(actor, "external-authenticator", identity_provider.silo.fleet);
-has_permission(actor: AuthenticatedActor, "list_identity_providers", identity_provider: IdentityProvider)
```

Comment:
I don't think this rule could have been used before because the only references to the `Action::ListIdentityProviders` action were on the Silo, not the IdentityProvider.